edn/src/edn.rustpeg

```diff
@@ -76,25 +76,22 @@ namespace_separator = "/"
 symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>]
 symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>]
 
-symbol_namespace = symbol_char_initial+ (namespace_divider symbol_char_subsequent+)*
+symbol_namespace = symbol_char_initial+ symbol_char_subsequent* (namespace_divider symbol_char_subsequent+)*
```

Comment:
It definitely needs `*` for symbol_char_subsequent, because the next thing in the production is a namespace divider.

It doesn't matter whether the initial `symbol_char_initial` is a `+` or not, because `s_c_i` is a subset of `s_c_s`.